.github/workflows/build.yml

```diff
@@ -2,19 +2,19 @@ name: build
 on: [push, pull_request]
 jobs:
   linters:
-    name: linters
+    name: linters / on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     timeout-minutes: 10
 
     strategy:
       matrix:
-        os: [ubuntu-18.04]  # TODO: pass linters on 'macos-10.15' 'windows-2019'
+        os: [ubuntu-18.04, macos-10.15]  # TODO: pass linters on 'windows-2019'
 
     steps:
       - name: Install Go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.13.11'
+          go-version: '1.14'
```

Comment:
a lot of groups are skipping 1.14 due to known issues with EINTR https://golang.org/doc/go1.14#runtime

Reply:
did kubernetes move up to 1.14?

Reply:
thx for the 1/1 discussion @hickeyma ...per discussion suggest following k8s model of using 1.13 for vendor and 1.13 and/or 1.14 for build/test/support

Reply:
Yes, Kubernetes moved up in: https://github.com/kubernetes/kubernetes/pull/92438. Thanks for heads up on the issue in Go 1.14.

Thanks for the discussion @mikebrow, it was very enlightening. I have raised #639 to cover Go 1.14 support. In the meantime, I am going to revert to 1.13 for this PR.